care/facility/tasks/ai_scribe.py

```diff
@@ -0,0 +1,138 @@
+    ai_form_fills = AIFormFill.objects.filter(
+        external_id=external_id, status=AIFormFill.Status.READY
+    )
+
+    for form in ai_form_fills:
+        # Skip forms without audio files
+        if not form.audio_file_ids:
+            logger.warning(f"AI form fill {form.external_id} has no audio files")
+            continue
+
+        logger.info(f"Processing AI form fill {form.external_id}")
+
+        audio_file_urls = []
+        transcript = ""
+
+        # Get the audio file URLs
+        for audio_file_id in form.audio_file_ids:
+            audio_file = FileUpload.objects.get(external_id=audio_file_id)
+            audio_file_urls.append(audio_file.read_signed_url())
+            logger.info(f"Audio file URL: {audio_file_urls[-1]}")
+
+        try:
+            # Update status to GENERATING_TRANSCRIPT
+            logger.info(f"Generating transcript for AI form fill {form.external_id}")
+            form.status = AIFormFill.Status.GENERATING_TRANSCRIPT
+            form.save()
+
+            if not form.transcript:
+                # Use Ayushma to generate transcript from the audio file
+                transcript = ""
+                for audio_file_url in audio_file_urls:
+                    audio_file_data = requests.request("GET", audio_file_url).content
+
+                    response = requests.request(
+                        "POST",
+                        f"{settings.AYUSHMA_ENDPOINT}/api/chats/transcribe",
+                        headers={"X-Api-Key": settings.AYUSHMA_API_KEY},
+                        data={"language": "en", "engine": "whisper"},
+                        files={"audio": ("file.mp3", audio_file_data)},
+                    )
+
+                    transcript += response.json()["transcript"]
+
+                    logger.info(f"Transcript: {transcript}")
+
+                # Save the transcript to the form
+                form.transcript = transcript
+            else:
+                transcript = form.transcript
+
+            # Update status to GENERATING_AI_RESPONSE
+            logger.info(f"Generating AI response for AI form fill {form.external_id}")
+            form.status = AIFormFill.Status.GENERATING_AI_RESPONSE
+            form.save()
+
+            # Process the transcript with Ayushma
+            ai_response = requests.request(
+                "POST",
+                f"{settings.AYUSHMA_ENDPOINT}/api/chats/completion",
+                headers={"X-Api-Key": settings.AYUSHMA_API_KEY},
+                json={
+                    "task": "ai_form_fill",
+                    "messages": [
+                        {
+                            "role": "system",
+                            "content": prompt_1,
+                        },
+                        {
+                            "role": "system",
+                            "content": prompt_2.replace(
+                                "{form_schema}", json.dumps(form.form_data, indent=2)
```

Comment:
string formatting would be more appropriate for this case but, we are just adding something at the end we can just append the string right?